src/v/storage/tests/storage_test_fixture.h

```diff
@@ -168,6 +168,7 @@ class storage_test_fixture : public ::testing::Test {
       ss::shared_ptr<storage::log> log,
       int appends,
       model::term_id term = model::term_id(0),
+      std::optional<model::timestamp> ts_override = std::nullopt,
```

Comment:
about one or two parameters ago I would have suggested to make this a struct argument so callsites get named parameters and it's more readable. Something to consider for a followup or future changes.